src/components/checkbox/demoBasicUsage/index.html

```diff
@@ -43,6 +43,18 @@
             Checkbox (md-primary): No Ink
           </md-checkbox>
         </div>
+        <div flex-xs flex="50">
+          <md-checkbox md-indeterminant
+            aria-label="Checkbox Indeterminate" class="md-primary">
+            Checkbox: Indeterminate
+          </md-checkbox>
+        </div>
+        <div flex-xs flex="50">
+          <md-checkbox md-indeterminant aria-label="Checkbox Disabled Indeterminate"
+            ng-disabled="true" class="md-primary">
```

Comment:
nit: indent to line up with md-indeterminate or indent 4 spaces instead of 2 to separate attributes from the checkbox content. (Ditto above)